apps/skolplattformen-app/components/children.component.tsx

```diff
@@ -30,7 +30,7 @@ import { RootStackParamList } from './navigation.component'
 
 const colors = ['primary', 'success', 'info', 'warning', 'danger']
 
-export const childenRouteOptions =
+export const childrenRouteOptions =
```

Comment:
`child`+`en` is actually a base plural rule for old English, but I don't think this is designed to be that.